.ci/bump-version.sh

```diff
@@ -10,7 +10,7 @@ docker run --rm -t \
   -e HOME=/tmp \
   -e AGENT_VERSION="${AGENT_VERSION}" \
   -v "$(pwd)/opbeans-dotnet:/app" \
-  mcr.microsoft.com/dotnet/core/sdk:2.2 /bin/sh -c "
+  mcr.microsoft.com/dotnet/core/sdk:3.1 /bin/sh -c "
```

Comment:
I think this will be always a 🐔 🥚 problem.

If we update opbeans-dotnet to run on a newer .NET version then we'll also need to update this here - and similarly to the current situation I'm afraid we'll forget about this.

All we need to do here is to update the agent version [here](https://github.com/elastic/opbeans-dotnet/blob/master/opbeans-dotnet/opbeans-dotnet.csproj#L11), right?

Reply:
We can potentially end to that problem if releasing from a previous version :/ 

> All we need to do here is to update the agent version here, right?

The `.ci/bump-version.sh` also updates the docker metadata, so hopefully the dotnet CI release automation should be the one driving this and we should not need any manual actions

Reply:
> If we update opbeans-dotnet to run on a newer .NET version then we'll also need to update this here - and similarly to the current situation I'm afraid we'll forget about this.

Indeed, we need to find another way, any ideas you have in mind?

Reply:
One thing that came to my mind is to rely on the zero code change agent injection. With that we don't need a direct reference in `opbenas-dotnet` to the `Elastic.Apm.*` package, therefore we don't need to execute this step.

But that'll include some other work, like fetching the agent binaries and setting an environment variable, so that'd work a bit differently.

Reply:
I see, uhm, how do you want to proceed? Raise an issue with the proposal or something else?

Regarding this change, if no issue, I'd like to merge it to move forward with the missing agent release in this project :)

Reply:
> I see, uhm, how do you want to proceed? Raise an issue with the proposal or something else?

I opened an issue

> Regarding this change, if no issue, I'd like to merge it to move forward with the missing agent release in this project :)

I agree with you - for now this is a totally acceptable solution.  👍